pkg/util/throughput/throughput.go

```diff
@@ -85,6 +91,12 @@ func (tc *throughputCalc) run() {
 			throughput = append(throughput, []interface{}{"metric-metadata/sec", int(metadataRate)}...)
 		}
 
+		duplicateSamplesRate := tc.duplicateSamples.Rate()
```

Comment:
I am not confident of printing this as regular log. Duplicates are part of "warn" which is why we had the previous way of logging duplicates via a Rate controlled log.Warn

Having this part of throughput seems like duplicates is a feature :)

Maybe this is not a big deal. If anyone else has other thoughts, then please chime in. Otherwise I will just approve.

Reply:
Yeah, it is not a feature, but when promscale users shares a log snippet we can easily find whether they have duplicate ingestion issue without going through the full log. Also dupicate message won't be append during a normal case!